business-registry-dashboard/app/pages/[encodedOrgId]/affiliationInvitation/acceptToken/[token].vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { useRoute } from 'vue-router'
+
+const { t } = useI18n()
+const accountStore = useConnectAccountStore()
+const affStore = useAffiliationsStore()
+const brdModal = useBrdModals()
+const route = useRoute()
+const toast = useToast()
+
+// Token parsing
+const parseToken = (encodedToken: string): AffiliationToken => {
+  try {
+    const tokenObject = encodedToken.split('.')[0]
```

Comment:
The token is multiple parts. In doing this, we grab the part that we actually need and decode it to get an object that looks like the interface.